### PR TITLE
Fix verification prefix name bug

### PR DESCRIPTION
**Pull request by keyboardDrummer**
### Changes
- Fix running verifiable symbols when using prefix name modules
- Fix requesting document symbols when using prefix name modules

### Testing
- Add test `VerifyPositionFromFile1AndUriFromFile2WherePrefixModuleIsInBothFiles`
- Add more assertions to test `LoadsSymbolsInPrefixModule`

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs [hunk 1]
```diff
@@ -49,6 +49,38 @@ public class VerificationStatusTest : ClientBasedLanguageServerTest {
...
+    });
+    var directory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+    var documentItem1 = CreateTestDocument(source, Path.Combine(directory, "A.dfy"));
+    await client.OpenDocumentAndWaitAsync(documentItem1, CancellationToken);
+    var documentItem2 = CreateTestDocument(source2, Path.Combine(directory, "B.dfy"));
+    await client.OpenDocumentAndWaitAsync(documentItem2, CancellationToken);
+    await AssertNoDiagnosticsAreComing(CancellationToken);
+    var fooInFileOnePosition = new Position(1, 9);
+    var runSuccess = await client.RunSymbolVerification(documentItem2, fooInFileOnePosition, CancellationToken);
+    Assert.False(runSuccess);
```
Comment (fabiomadge) [inline Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs:81]: Don't we expect this to succeed?

File: Source/DafnyCore/AST/Modules/ModuleDefinition.cs [hunk 2]
```diff
@@ -433,7 +435,7 @@ public class ModuleDefinition : RangeNode, IDeclarationOrUsage, IAttributeBearin
     return true;
   }
 
-  public void ProcessPrefixNamedModules() {
+  public void ProcessPrefixNamedModules(bool setRange) {
```
Comment (fabiomadge) [inline Source/DafnyCore/AST/Modules/ModuleDefinition.cs:438]: Do you actually use this argument anywhere?

**Review by fabiomadge — COMMENTED** [review]
Two quick questions.

File: Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs (hunk 1 above)
Comment (keyboardDrummer) [reply Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs]: Updated the name

File: Source/DafnyCore/AST/Modules/ModuleDefinition.cs (hunk 2 above)
Comment (keyboardDrummer) [reply Source/DafnyCore/AST/Modules/ModuleDefinition.cs]: It's not used. Sorry, I should have removed it already.

**Review by fabiomadge — APPROVED** [review]

**Pull request merged**